pkgs/development/python-modules/reproject/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, astropy
+, astropy-healpix
+, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "reproject";
+  version = "0.4";
+
+  doCheck = false; # tests require pytest-astropy
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "dbbb18a8b211292c7ce61121b8538fc279540337be1c05cabc7570c5aca6d734";
+  };
+
+  propagatedBuildInputs = [ numpy astropy astropy-healpix scipy ];
+
+  meta = with lib; {
+    description = "Reproject astronomical images";
+    homepage = https://reproject.readthedocs.io;
+    license = licenses.bsdOriginal;
```

Comment:
https://github.com/astropy/reproject/blob/master/LICENSE.md is bsd3

Reply:
Weird, it's listed as bsd original on [Pypi](https://pypi.org/project/reproject/).

Reply:
"BSD" can mean anything :)